### DISCUSSION TITLE
phase 3 freezing

### DISCUSSION THREAD

**Discussion by kajtek17r** [swar/Swar-Chia-Plot-Manager#988] [General]
Ok, i held enough with this. And this is the most frustrating thing that has happend to me last 2 weeks. So, in phase 3 some plots (usually those which are close to finish) are getting stuck for no reason. I have tried many configs, but none of them gave me result. I have tried to find something similar in the past issues and discussions but nothing is comparable.

And now some screns and logs.

<img width="576" alt="Screenshot_6" src="https://user-images.githubusercontent.com/84998581/120050441-a6cd7780-c01d-11eb-9054-8632668fd1c3.png">
<img width="427" alt="Screenshot_1" src="https://user-images.githubusercontent.com/84998581/120050483-c1075580-c01d-11eb-95ff-9ab49bfa81a3.png">
[inland_2021-05-27_08_28_27_999459.log](https://github.com/swar/Swar-Chia-Plot-Manager/files/6563687/inland_2021-05-27_08_28_27_999459.log)
[inland_2021-05-27_12_30_37_989836.log](https://github.com/swar/Swar-Chia-Plot-Manager/files/6563688/inland_2021-05-27_12_30_37_989836.log)

If u need any more info i will give you anything just please tell me what is going on with these freezes
.
My specs:

i9-10850K
1x2TB NVME wd 850
32 GB RAM 3600
hdd seagate 8tb

 Now my CURRENT settings:
  - name: inland
    max_plots: 999
    farmer_public_key:
    pool_public_key:
    temporary_directory: C:\Chia
    temporary2_directory:
    destination_directory:
      - G:\Chia3
      - H:\Chia4
    size: 32
    bitfield: true
    threads: 4
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 7
    max_concurrent_with_start_early: 8
    initial_delay_minutes: 0
    stagger_minutes: 40
    max_for_phase_1: 3
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19 ]

**Comment by papa41652** [comment]
question what inland are you using pro, premium or platinum?

**Reply by kajtek17r** [reply]
It's just a job name from default template. I am using WD Black SN850 2TB

**Comment by Nilmot** [comment]
I had the same problem, it was the RAM OC remove any ram oc and give it a try.

**Comment by Chilli71** [comment]
Did you have a look in the plot log files if they give some error messages?

**Reply by kajtek17r** [reply]
yea, i have checked every log. none info about errors.

**Comment by kajtek17r** [comment]
ok, problem solved. 2 days ago i decreased memory usage from 4000 to 3400 per one plot and everything is smooth now. didn't change OC etc. just decreased RAM for each plot. @swar we can close the thread

**Reply by Chilli71** [reply]
Actually, memory was my first thought. But on the screenshot that you provided enough memory was available.
If it is the memory, maybe you make one job less to be on the safe side.

**Reply by kajtek17r** [reply]
i didn't even suspect it could be memory, especailly it was always around 60-70% used. i'd rather go with one more job and less memory for each, not sure which is better. anyway thanks for your contribution :)

**Reply by Chilli71** [reply]
Actually, I meant one job less and less memory per job. Unless you have a pure plotting machine that you don't us for anything else.
If this is this is your daily use computer you risk that everything crashes again if you need a bit memory for other programs.
Nor sure if you are using windows - my windows plotter reacts much more allergic on low memory than the linux one does.
